contracts/Libraries/BiMap.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.5.0;
+
+
+library BiMap {
+    struct Data { 
+        mapping(uint16 => address) idToAddress;
+        mapping(address => uint16) addressToId;
+    }
+
+    function hasId(Data storage self, uint16 id) public view returns (bool) {
```

Comment:
Naming-wise we are kind of committing here to a ID->Address BiMap. I wonder if we should then just reflect this in the library name.
Alternatively we could call this a bit more generic (`hasLeftKey`, `hasRightKey`)

Reply:
I wouldn't want to discriminate between left and right in a Bijective map. Maybe better reflect this in the name